llvm/test/Transforms/LoopVectorize/AArch64/partial-reduce-dot-product-neon.ll

```diff
@@ -2,6 +2,7 @@
 ; RUN: opt -passes=loop-vectorize -enable-epilogue-vectorization=false -mattr=+neon,+dotprod -force-vector-interleave=1 -S < %s | FileCheck %s --check-prefixes=CHECK-INTERLEAVE1
 ; RUN: opt -passes=loop-vectorize -enable-epilogue-vectorization=false -mattr=+neon,+dotprod -S < %s | FileCheck %s --check-prefixes=CHECK-INTERLEAVED
 ; RUN: opt -passes=loop-vectorize -enable-epilogue-vectorization=false -mattr=+neon,+dotprod -force-vector-interleave=1 -vectorizer-maximize-bandwidth -S < %s | FileCheck %s --check-prefixes=CHECK-MAXBW
+; RUN: opt -passes=loop-vectorize -debug-only=loop-vectorize --disable-output -S < %s 2>&1 | FileCheck %s --check-prefix=CHECK-REGS
```

Comment:
This now needs a `REQUIRES: asserts` to guard it for the `-debug-only=`.

Reply:
Done.

Reply:
This is just checkoing the debug output for a single function, right? Might be better to move to the existing `llvm/test/Transforms/LoopVectorize/AArch64/reg-usage.ll` if possible

Reply:
That makes sense, done.